lib/slack/real_time/socket.rb

```diff
@@ -30,6 +34,13 @@ def connect!
         connect
         logger.debug("#{self.class}##{__method__}") { driver.class }
 
+        @alive = true
+
+        driver.on :message do |event|
+          event_data = JSON.parse(event.data)
```

Comment:
I am not sure we do, but do we need to handle garbage from the server here?

Reply:
It's a good idea, though I'd recommend handling that in a separate PR. One where `Slack::Messages::Message` is used and updating it so that it would take care of any parsing errors that may occur.